addons/math-in-inputs/addon.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Allow math in number inputs",
+  "description": "Use math symbols (*, /, -, +, etc) in number inputs and have them automatically solve.",
```

These should probably be phrased more similarly to other addons.

```suggestion
  "name": "Mathematical expressions in number inputs",
  "description": "Evaluates mathematical expressions (e.g. 2+4*3) in number inputs in the editor automatically.",
```